source/includes/fact-match-dump-restore-versions.rst

```diff
@@ -1,9 +1,28 @@
 When using :binary:`~bin.mongorestore` to load data files created by
-:binary:`~bin.mongodump`, be sure that you are restoring to the *same
-major version* of the MongoDB Server that the files were created from.
-For example, if your dump was created from a MongoDB Server running
-version ``4.4.x``, be sure that the MongoDB Server you are restoring to
-is also running version ``4.4.x``.
+:binary:`~bin.mongodump`, the MongoDB versions of your source and 
+destination deployments must be either:
+
+- The same major version.
+- The same feature compatibility version. 
+
+For example, if your dump was created from a MongoDB deployment running
+version ``4.4``, the MongoDB deployment you restore to must also run 
+version ``4.4`` or have its FCV level set to ``4.4``.
+
+For information on changing feature compatibility versions, 
```

Comment:
I should've spotted this in the first pass, but this seems like an instance where we can speak more to the user's action:

To change your deployment's feature compatibility version, see (link).